std_execution.bs

```diff
@@ -4126,19 +4128,17 @@ from the operation before the operation completes.
 
     1. <i>Effects:</i> Waits until one of the following conditions is `true`:
 
-        * <i>count</i> is `0` and <i>state</i> is <i>finishing</i>, in which case `pop_front()` returns `nullptr`; or
+        * <i>count</i> is `0` and <i>state</i> is <i>finishing</i>, in which case `pop_front` returns `nullptr`; or
 
-        * <i>count</i> is greater than `0`, in which case `pop_front()` removes an item from the front of the queue, decrements <i>count</i> by `1`, and returns the item.
-
-    2. <i>Synchronization:</i> This function is thread-safe with respect to `push_back()` and `finish()`. Removing a queue item and decrementing <i>count</i> are executed atomically.
+        * <i>count</i> is greater than `0`, in which case an item is removed from the front of the queue, <i>count</i> is decremented by `1`, and the removed item is returned.
 
     <pre highlight="c++">
     void run_loop::push_back(<i>run-loop-opstate-base</i>* item);
     </pre>
 
-    1. <i>Effects:</i> Adds `item` to the back of the queue and increments <i>count</i> by `1`.
+    1. <i>Effects:</i> Atomically adds `item` to the back of the queue and increments <i>count</i> by `1`.
 
-    2. <i>Synchronization:</i> This function is thread-safe with respect to `pop_front()` and `finish()`. Adding a queue item and incrementing the <i>count</i> are executed atomically.
+    2. <i>Synchronization:</i> This operation synchronizes with the `pop_front` operation that obtains `item`.
```

Comment:
Do you need to say something about not introducing data races?

Reply:
Nope - that's covered by some wording in the front matter, similar to what we do for barrier (https://eel.is/c++draft/thread.barrier#class-4).